pagopa/io-app
# feat: [PE-630,PE-758] CGN partner search screen

## Short description
Added a partner's search new screen.
Refer to screens in https://pagopa.atlassian.net/browse/PE-630 description.
See https://pagopa.atlassian.net/browse/PE-758 for no search results screen

## List of changes proposed in this pull request
- add labels
- remove search functionality in discount list by operators
- add new search page

## How to test

To test with dev-api-server this PR is needed https://github.com/pagopa/io-dev-api-server/pull/432
I did not made this PR depend on it since it is testable using other backend environments

Going to the new search screen option 1:
- login into the app
- go to CGN card trough wallet (add it if necessary)
- press discover opportunities
- press magnifying icon in the top bar

Going to the new search screen option 2:
- login into the app
- go to CGN card trough wallet (add it if necessary)
- press discover opportunities
- go to any category
- press magnifying icon in the top bar

Testing the search screen

- Go to the new search screen
- search for partner
- check if you see the results
- press on a result and check if it opens the correct screen

| overview |
| - |
| <video src="https://github.com/user-attachments/assets/ded627b4-a068-43c3-8a15-c3b8af2ae998"/> |
